app/views/katello/subscription_mailer/subscription_expiry.text.erb

```diff
@@ -0,0 +1,13 @@
+<%= _("Subscriptions expiring soon") %>
+
+<% @pools.each do |pool| %>
+  <%= pool %>:
+  <%= _("Subscription Name") %>: <%= pool.subscription.name %>
+  <%= _("Organization") %>:      <%= Organization.find_by_id(pool.organization_id).name %>
+  <%= _("Type") %>:              <%= pool.pool_type %>
+  <%= _("Quantity") %>:          <%= pool.quantity %>
+  <%= _("SKU") %>:               <%= pool.subscription.cp_id %>
+  <%= _("Contract") %>:          <%= pool.contract_number %>
+  <%= _("Account Start") %>:     <%= pool.start_date %>
```

Comment:
_The email notification will contain following information about expiring subscriptions:
subscription name, satellite organization, type, quantity, SKU, contract, __account__, start, end (expiry)_

Seems like we're missing a column for Account number.